src/c++/perf_analyzer/report_writer.cc

```diff
@@ -51,7 +51,7 @@ ReportWriter::Create(
 
 ReportWriter::ReportWriter(
     const std::string& filename, const bool target_concurrency,
-    std::vector<pa::PerfStatus>& summary, const bool verbose_csv,
+    std::vector<pa::PerfStatus> summary, const bool verbose_csv,
```

Comment:
const &?

Reply:
This one is the copy.
I updated the other one to be a  const & now.

Reply:
I think you can still do const& and `summary_` to be `std::vector<pa::PerfStatus> summary`  then the copy will still work

Reply:
You are totally right. Thanks